assets/js/admin/wc-product-import.js

```diff
@@ -40,7 +42,7 @@
 				position      : $this.position,
 				mapping       : $this.mapping,
 				file          : $this.file,
-				skip_existing : $this.skip_existing,
+				update_existing : $this.update_existing,
```

Comment:
Alignment got a little off in some places in this file and class-wc-admin-importers.php where you've changed 'skip_existing' to 'update_existing'. Not a big deal, just being thorough in my review.